symfit/core/objectives.py

```diff
@@ -296,5 +429,14 @@ def eval_jacobian(self, ordered_parameters=[], **parameters):
                 ordered_parameters, **parameters
             )
             return np.array(evaluated_jac[0])
+        else:
+            return None
+
+    def eval_hessian(self, ordered_parameters=[], **parameters):
+        if hasattr(self.model, 'eval_hessian'):
+            evaluated_hess = super(MinimizeModel, self).eval_hessian(
+                ordered_parameters, **parameters
+            )
+            return np.array(evaluated_hess[0])
```

Comment:
Do we check anywhere that is is in fact a single component model?

Reply:
From the docstring:
```python
    """
    Objective to use when the model itself is the quantity that should be
    minimized. This is only supported for scalar models.
    """
```
And that rule is enforced in __init__ ;)